### PR TITLE
test delete files in branch after branch deleted

### PR DESCRIPTION
**Pull request by romanbracinik**
Jira: SOX-126
KBC: https://github.com/keboola/connection/pull/4533

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

File: tests/Backend/SOX/BranchFileStorageTest.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    }
+
+    public function testDeleteBranchDeleteFiles(): void
+    {
+        $newBranch = $this->branches->createBranch($this->generateDescriptionForTestObject());
+
+        $branchClient = $this->getBranchAwareClient($newBranch['id'], [
+            'token' => STORAGE_API_DEVELOPER_TOKEN,
+            'url' => STORAGE_API_URL,
+        ]);
```
Comment (zajca) [inline tests/Backend/SOX/BranchFileStorageTest.php:37]: možeš teď používat:
```suggestion
        $branchClient = $this->getDeveloperStorageApiClient()->getBranchAwareClient($newBranch['id']);
```

**Review by zajca — APPROVED** [review]

File: tests/Backend/SOX/BranchFileStorageTest.php (hunk 1 above)
Comment (romanbracinik) [reply tests/Backend/SOX/BranchFileStorageTest.php]: Ha diky :)

File: tests/Backend/SOX/BranchFileStorageTest.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        $this->assertEquals(file_get_contents($filePath), $object['Body']);
+
+        $this->branches->deleteBranch($newBranch['id']);
+
+        try {
+            $s3Client->getObject([
+                'Bucket' => $file['s3Path']['bucket'],
+                'Key' => $file['s3Path']['key'],
+            ]);
+            $this->fail('File should not exist');
```
Comment (romanbracinik) [inline tests/Backend/SOX/BranchFileStorageTest.php:63]: @zajca ja som mal este necommitnutu zmenu tak pridavam

**Review by zajca — APPROVED** [review]

File: tests/Backend/SOX/BranchFileStorageTest.php (hunk 2 above)
Comment (zajca) [reply tests/Backend/SOX/BranchFileStorageTest.php:63]: :see_no_evil: to sem si asi měl všimnůt aj :D

**Comment by romanbracinik** [comment]
rebase

**Pull request merged**